src/main/java/com/softserve/messaging/config/KafkaConfiguration.java

```diff
@@ -26,24 +26,28 @@
     private String groupId;
 
```

Comment:
remove this file, all the properties you use can be specified using application.yml

Reply:
also, to fix the issue with types, you will need to specify property `spring.kafka.consumer.properties.spring.json.type.mapping`

Reply:
i'm very grateful for this advice)
fixed